src/Rector/Contrib/Sylius/Review/ReplaceCreateMethodWithoutReviewerRector.php

```diff
@@ -0,0 +1,73 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector\Contrib\Sylius\Review;
+
+use PhpParser\Node;
+use Rector\NodeAnalyzer\MethodArgumentAnalyzer;
+use Rector\NodeAnalyzer\MethodCallAnalyzer;
+use Rector\NodeChanger\IdentifierRenamer;
+use Rector\Rector\AbstractRector;
+
+/**
+ * @author Eduard Barnáš <barnas@neoweb.sk>
+ */
```

Comment:
Coding policy of this is to keep authorship via `git blame` and https://github.com/rectorphp/rector/graphs/contributors
It's more precise than annotations and here it would only duplicate information

Could you remove it?